File: data/users/admin/perm_missing_elem/sharp/perm_missing_elem_test_example.cs
using System;

namespace perm_missing_elem
{ 
	class MainClass
	{
		public static void Main (string[] args)
		{
			int i;
			int result;
			// test_struct[] tests = new test_struct[COUNT_EXAMPLE_TEST]; 
			test_struct[] tests = new test_struct[1]; 
	
			tests[0].SetValueExample( 
				new int[]{2, 3, 1, 5},
				4, 
				4
			);
			
			Solution s = new Solution();
			
			for (i = 0; i < 1; ++i) {
				result = s.solution(tests[i].A);

				Console.Write("| ");
				
				Console.Write("Example test: (");
				General.print_arr(tests[i].A, tests[i].N);
				Console.Write(")");
				
				Console.Write(" | ");

				/* функция проверки результата */
				if ( tests[i].R == result )
					Console.Write("OK");
				else {
					Console.Write("WRONG ANSWER (got {0}", result);
					Console.Write(" expected {0}", tests[i].R);
					Console.Write(")");
					Console.Write(" | ");
				}

				Console.Write(" | ");
				Console.Write("\n");
				Console.Write("\n");
			}
		}
	}
}

File: data/users/admin/perm_missing_elem/c/perm_missing_elem_test_example.c
#include <stdio.h>
#include "perm_missing_elem.h"

int main()
{
	int    i;
	int    result;
	struct test_example tests[COUNT_EXAMPLE_TEST];

	tests[0] = (struct test_example){
		(int []){2, 3, 1, 5},
		4,
		4
	};

	for (i = 0; i < COUNT_EXAMPLE_TEST; ++i) {
		result = solution(tests[i].A, tests[i].N);
		
		printf ("| ");
		
		printf ("Example test: (");
		print_arr(tests[i].A, tests[i].N);
		printf (")");
		
		printf (" | ");

		/* функция проверки результата */
		if ( tests[i].R == result )
			printf ("OK");
		else {
			printf ("WRONG ANSWER (got %d", result);
			printf (" expected %d", tests[i].R);
			printf (")");
			printf (" | ");
		}
		
		printf (" | ");
		printf ("\n");
		printf ("\n");
	}
	return 0;
}
